lib/cli.coffee

```diff
@@ -0,0 +1,41 @@
+async = require 'async'
+minimist = require 'minimist'
+MychaInstaller = require './mycha_installer'
+MochaRunner = require './mocha_runner'
+prompt = require 'prompt'
+
+
+argv = process.argv.slice 2
+cwd = process.cwd()
+
+
+exitWithError = (err) ->
+  console.error err
+  process.exit 1
+
+
+getTestHelperPath = (done) ->
+  prompt.start()
+  prompt.get [
+    description: 'Default test helper path',
+    name: 'testHelperPath'
+    required: true
+  ], (err, {testHelperPath} = {}) ->
+    if err then return done err
+    done null, testHelperPath
+
+
+installMycha = (testHelperPath, done) ->
+  installer = new MychaInstaller {cwd, testHelperPath}
+  installer.install done
+
+
+if minimist(argv)._[0] is 'install'
```

Comment:
`nconf` uses `optimist` which the maintainer has deprecated because he uses `minimist`. 

Removed from here as what I'm doing is so simple I don't need minimist but still use it in ArgumentBuilder to determine if there are any files specified.